leftwm/src/config/mod.rs

```diff
@@ -652,6 +664,20 @@ impl Config {
     }
 }
 
+fn from_regex<'de, D: Deserializer<'de>>(deserializer: D) -> Result<Option<Regex>, D::Error> {
```

Comment:
I guess this is where I'd put a comment about the `re2` compliance

Reply:
@VuiMuich Done